pkg/controller/storagecluster/storagecluster.go

```diff
@@ -30,6 +30,7 @@ import (
 
 	"github.com/hashicorp/go-version"
 	"github.com/libopenstorage/operator/drivers/storage"
+	pxutil "github.com/libopenstorage/operator/drivers/storage/portworx/util"
```

Comment:
Let's not add/reference driver specific code from the controllers. The `IsPartialSecretRef` function can be moved to the package`pkg/util` as it has nothing to do with portworx driver.